app/src/main/java/com/stevesoltys/backup/transport/component/provider/ContentProviderRestoreComponent.java

```diff
@@ -64,14 +81,16 @@ public int startRestore(long token, PackageInfo[] packages) {
         restoreState.setPackages(packages);
         restoreState.setPackageIndex(-1);
 
-        if (configuration.getPassword() != null && !configuration.getPassword().isEmpty()) {
+        String password = requireNonNull(this.password);
+
+        if (!password.isEmpty()) {
             try {
                 ParcelFileDescriptor inputFileDescriptor = buildInputFileDescriptor();
                 ZipInputStream inputStream = buildInputStream(inputFileDescriptor);
                 seekToEntry(inputStream, ContentProviderBackupConstants.SALT_FILE_PATH);
```

Comment:
Could either remove `ContentProviderBackupConstants`, or migrate over any constants you think should be there. (your preference - either is fine for me)

Reply:
Unless this is the only shared constant between backups / restores? That would actually make sense for `ContentProviderBackupConstants`.

Reply:
I hadn't seen `ContentProviderBackupConstants` and agree that the constants should go there!